### DISCUSSION TITLE
Wallpaper with realtime local temperature and parallax.

### DISCUSSION THREAD

**Discussion by ViictorFidelis** [rocksdanister/lively#2117] [Show and tell]
I've been working on this for a while, just wanted to have a nice and simple background, did it for myself, but feel free to use it. :D

![image](https://github.com/rocksdanister/lively/assets/159702209/d52260f3-800f-4722-9cf5-f84b31bfbd97)

Download:
https://github.com/ViictorFidelis/Parallax-Wallpaper/releases/download/Wallpaper/Date.time.with.parallax.zip

Customizable options:
city
language
backgroundIMG

note: both city and language are necessary for the geolocation api.

**Comment by rocksdanister** [comment]
Good job, it looks amazing.

Looks like the file is not properly packaged though, I would create the wallpaper file within Lively by ... -> Share -> Save to file.

Then upload it in Github release page:
https://github.com/ViictorFidelis/Parallax-Wallpaper/releases
